runtime/doc/eval.txt

```diff
@@ -5832,6 +5833,9 @@ setqflist({list} [, {action}])				*setqflist()*
 		with the items from {list}. If {action} is not present or is
 		set to ' ', then a new list is created.
 
+		If {title} is given, it will be stored in the variable
+		|w:quickfix_title|.
+
```

Comment:
“If {title} is given it will be used as the {quickfix,location} window title available through |w:quickfix_title| after opening the window.” Title is stored even if there is no window available, so it is not correct to state that it is stored there. Also this specific phrase sounds like “setqflist() will do `let w:quickfix_title = {title}`” because when not said otherwise variable is expected to refer to the *current* window.